knowledge_repo/app/templates/base.html

```diff
@@ -114,7 +114,7 @@
         name: 'knowledge_posts',
         limit: 10,
         display: function (item) {
-          return item;
+          return item.title + " - " + item.author;
```

Comment:
Yeah I see now that you can still see the full highlight with title/author when toggling down on the options. This is fine.